src/platform/plugins/shared/dashboard/public/dashboard_api/types.ts

```diff
@@ -106,7 +107,7 @@ export type DashboardApi = CanExpandPanels &
   PublishesDataLoading &
   PublishesDataViews &
   PublishesDescription &
-  Pick<PublishesTitle, 'title$'> &
+  Pick<PublishesTitle, 'title$' | 'hideTitle$'> &
```

Comment:
Does the API no longer have `hideTitle$`? Do we not need this for the ability to hide all titles on the Dashboard?